internal/signalfx-agent/pkg/monitors/prometheusexporter/prometheus.go

```diff
@@ -45,13 +45,29 @@ type Config struct {
 	// (the default).
 	MetricPath string `yaml:"metricPath" default:"/metrics"`
 
+	// Control the log level to use if a scrape failure occurs when scraping
+	// a target. Modifying this configuration is useful for less stable
+	// targets. Only the debug, info, warn, and error log levels are supported.
+	ScrapeFailureLogLevel string `yaml:"scrapeFailureLogLevel" default:"error"`
```

Comment:
I was wondering the same about updating documentation and examples. Still looking around.

Reply:
`make docs` and the docs directory weren't ported from the agent project since the go dependencies were the initial concern. Taking some of their doc generation scripts and embedding them as generate directives for `README.md` generation would be my vote (in unrelated changes).

Reply:
Would support the `make docs` option, I can create a gh ticket and supporting materials for this if it is not already captured in another effort. 

Would support a similar approach where possible in the chart repo for docs automation.